vignettes/laminr.Rmd

```diff
@@ -41,9 +41,9 @@ lamin connect <owner>/<name>
 ```
 
 This instance acts as the default instance for everything that follows.
-Any new records or other changes will be added here.
+Any data and tracking information will be added to it.
 
-# Connect to the default instance
+# Start an R session
```

Comment:
I changed this before as I find "Start an R session" confusing as users already need to have an R session to run the commands

Reply:
But then we need another heading. We can call it "Start an analysis session in R".

The commands above are all on the terminal except the `install.packages` line.

Reply:
Or "Start your analysis"

Reply:
I'll make this fix.